server/utils.py

```diff
@@ -0,0 +1,7 @@
+import re
+
+regex = re.compile(r"([A-Za-z0-9]+[.-_])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})+")
```

Comment:
It's a very very bad naming for a  constant in utils

Reply:
My bad. Totally

Reply:
Fixed